rest_sensors.cpp

```diff
@@ -905,46 +905,58 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 }
                 else if (rid.suffix == RConfigOffset) // Signed integer
                 {
-                    data.integer = data.integer / 10;
```

Comment:
I think this breaks `config/offset` for all non Tuya devices?

Reply:
Yes, was the change too much, corrected.